diff --git a/node_modules/react-native-screens/android/build.gradle b/node_modules/react-native-screens/android/build.gradle
--- a/node_modules/react-native-screens/android/build.gradle
+++ b/node_modules/react-native-screens/android/build.gradle
@@ -206,7 +206,7 @@
 
 repositories {
     maven {
-        url "${reactNativeRootDir}/android"
+        url = "${reactNativeRootDir}/android"
     }
 
     mavenCentral()
